src/components/masthead/masthead.styles.ts

```diff
@@ -26,7 +26,7 @@ export const styles = StyleSheet.create({
     backgroundColor: '#000',
   },
   navToggle: {
-    color: global_Color_100.value,
+    color: global_Color_light_100.value,
```

Comment:
Not sure about using the light color because it sounds like a PatternFly bug? Thinking we should raise the issue withe the PatternFly folks so it can be resolved.

Reply:
@dlabrecq sure.
It was mentioned [here](https://github.com/project-koku/koku-ui/issues/72#issuecomment-415421688) and shown [there](https://docs.google.com/presentation/d/1Ye5Vhj3AqQStZINtqUJ4Y8-mzKX3qBWRmlMQdP-lnwY/edit#slide=id.g3d1ce4edfa_0_50)

Reply:
Ah, ok. It's just for mobile then.

The only potential issue is with the icon state. After selecting the times icon, and I adjust the screen size, it always shows the times icon. If I don't select that first, then the hamburger icon is always shown. 

Is that how it's intended to work? I was expecting to see the same hamburger icon at a particular screen size.

Reply:
![new gif](https://user-images.githubusercontent.com/17481322/44664360-074ca780-a9e1-11e8-9137-df30243f81ec.gif)